cms/templates/admin/cms/page/close_frame.html

```diff
@@ -19,13 +19,11 @@
     (function(Window) {
         // the dataBridge is used to access plugin information from different resources
         // Do NOT move this!!!
-        Window.CMS.API.Helpers.dataBridge = {
-            'plugin_id': '{{ plugin_id|unlocalize }}',
-            'plugin_type': '{% filter escapejs %}{{ type }}{% endfilter %}',
-            'plugin_name': '{% filter escapejs %}{{ name }}{% endfilter %}',
-            'plugin_icon': '{% filter escapejs %}{{ icon }}{% endfilter %}',
-            'plugin_desc': '{% filter escapejs %}{{ alt }}{% endfilter %}'
-        };
+        Window.CMS.API.Helpers.dataBridge = {{ plugin_data|safe }};
```

Comment:
|safe might cause problems if not correctly treated in the JS, are we sure this will not allow js injecting?